src/views/Player.vue

```diff
@@ -31,7 +31,7 @@
       :title="title"
       greeting="Hooray! Congrats on completing the quiz! 🎉"
       :numQuestionsAnswered="numQuestionsAnswered"
-      @restart-quiz="restartQuiz"
+      @last-question="toLastQuestion"
```

Comment:
sending it to last question is how we are implementing it, the scorecard doesn't know that. the scorecard is only emitting that the "go back" button has been clicked.

```suggestion
      @go-back="goToLastQuestion"
```